tokens/manage/identity.md

```diff
@@ -206,15 +206,24 @@ When requesting identity judgement, you'll need to provide the index of the regi
-    | [Registrar #0](https://forum.moonbeam.foundation/t/proposal-32-status-voting-identity-registrar-0/187){target=_blank} | Moonbeam Foundation | 0x031590D13434CC554f7257A89B2E0B10d67CCCBa |   0   |
+    |                                                               Registrar                                                                |                              Operator                              |                  Address                   | Index |
+    |:--------------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------:|:------------------------------------------:|:-----:|
+    |         [Registrar #0](https://forum.moonbeam.foundation/t/proposal-32-status-voting-identity-registrar-0/187){target=_blank}          | [Moonbeam Foundation](https://moonbeam.foundation/){target=_blank} | 0x031590D13434CC554f7257A89B2E0B10d67CCCBa |   0   |
+    | [Registrar #1](https://forum.moonbeam.foundation/t/referendum-125-status-passed-new-registrar-proposal-registrar-1/303){target=_blank} |         [Chevdor](https://www.chevdor.com/){target=_blank}         | 0x4aD549e07E96BaD335A8b99C8fd32e95EE538904 |   1   |
 
 === "Moonbase Alpha"
-    |                                      Registrar                                      |                      Operator                      |                  Address                   | Index |
-    |:-----------------------------------------------------------------------------------:|:--------------------------------------------------:|:------------------------------------------:|:-----:|
-    | [Registrar #1](https://www.chevdor.com/post/2020/07/14/reg-updates/){target=_blank} | [Chevdor](https://www.chevdor.com/){target=_blank} | 0x4aD549e07E96BaD335A8b99C8fd32e95EE538904 |   1   |
+    |                                                           Registrar                                                            |                              Operator                              |                  Address                   | Index |
+    |:------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------:|:------------------------------------------:|:-----:|
+    |                      [Registrar #1](https://www.chevdor.com/post/2020/07/14/reg-updates/){target=_blank}                       |         [Chevdor](https://www.chevdor.com/){target=_blank}         | 0x4aD549e07E96BaD335A8b99C8fd32e95EE538904 |   1   |
+    | [Moonbeam Foundation Registrar](https://moonbase.subscan.io/account/0xf2E866bcF1B8569C4CE4E10AF2775deA04ce9A47){target=_blank} | [Moonbeam Foundation](https://moonbeam.foundation/){target=_blank} | 0xf2E866bcF1B8569C4CE4E10AF2775deA04ce9A47 |   2   |
```

Comment:
used "Moonbeam Foundation Registrar" as opposed to "Registrar #2" because the on-chain identity is "Moonbeam Foundation Registrar" 🤷‍♀️

Reply:
I think the foundation is Registrar 0 on both networks so should be above Chevdor

Reply:
I asked internally who is Registrar 0 and who is that Registrar 2 in Moonbase

Reply:
ok let me know

Reply:
let's remove moonbase registrars (but chevdor) for now